cmd/utils/flags.go

```diff
@@ -534,10 +534,14 @@ var (
 		Value: whisper.DefaultMinimumPoW,
 	}
 
-	//Sharding Settings
+	// Sharding Settings
 	DepositFlag = cli.BoolFlag{
 		Name:  "deposit",
-		Usage: "To become a notary with your sharding client, " + new(big.Int).Div(sharding.NotaryDeposit, new(big.Int).Exp(big.NewInt(10), big.NewInt(18), nil)).String() + " ETH will be deposited into SMC",
+		Usage: "To become a notary in a sharding node, " + new(big.Int).Div(sharding.NotaryDeposit, new(big.Int).Exp(big.NewInt(10), big.NewInt(18), nil)).String() + " ETH will be deposited into SMC",
+	}
+	ProtocolFlag = cli.StringFlag{
+		Name:  "protocol",
```

Comment:
Is protocol the correct flag name? What about `actor` or `role`?

Reply:
Actor sounds good to me